themes/beaver/assets/css/use-cases-dynamic.css

```diff
@@ -0,0 +1,150 @@
+.use-cases>.heading {
+    color: #1a8cff;
+    font-weight: 600;
+    text-align: center;
+    margin-bottom: 30px;
+}
+.use-cases .subheading {
+    font-size: 60px;
+    color: #121212;
+    margin-left: 200px;
+    margin-right: 200px;
+    text-align: center;
+}
+
+@media(max-width: 1115px) {
+    .use-cases .subheading {
+        margin-left:50px;
+        margin-right: 50px
+    }
+}
+
+@media(max-width: 860px) {
+    .use-cases .subheading {
+        margin-left:0;
+        margin-right: 0;
+        margin-top: 10px
+    }
+}
+
+.use-cases .description {
+    margin-left: 300px;
+    margin-right: 300px;
+    margin-top: 20px;
+    text-align: center;
+}
+
+@media(max-width: 1115px) {
+    .use-cases .description {
+        margin-left:50px;
+        margin-right: 50px
+    }
+}
+
+@media(max-width: 860px) {
+    .use-cases .description {
+        margin-left: 0;
+        margin-right: 0;
+        margin-top: 15px;
+    }
+    .use-cases .tab-title {
+        font-size: 16px;
+    }
+}
+
+.use-cases .tabs-container {
+    flex-direction: column;
+}
+.use-cases .tab-title {
+    background-color: #f5f6f8;
+    color: #121212;
+    border: none;
+    border-radius: 14px;
+    padding: 28px 35px;
+    font-size: 20px;
+    font-weight: 700;
+    cursor: pointer;
+    outline: none;
```

Comment:
I guess we should not remove that Please see here https://gist.github.com/guilhermesimoes/4595862